clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/deploy/handlers/TitusDeployHandler.groovy

```diff
@@ -144,6 +144,12 @@ class TitusDeployHandler implements DeployHandler<TitusDeployDescription> {
           }
           sourceJob.labels.each { k, v -> description.labels.put(k, v) }
         }
+        if (!description.containerAttributes || description.containerAttributes.isEmpty()) {
+          if (!description.containerAttributes) {
+            description.containerAttributes = [:]
+          }
+          sourceJob.containerAttributes.each { k, v -> description.containerAttributes.put(k, v) }
```

Comment:
I think this should be outside the parent `if` so that it is run if there are containerAttributes (not just if it's an empty map)

Reply:
we're copying from the source asg here